packages/react/src/components/Tag/Tag.js

```diff
@@ -66,6 +66,7 @@ const Tag = ({
       <button
         className={`${prefix}--tag__close-icon`}
         onClick={handleClose}
+        disabled={disabled}
```

Comment:
Should we also remove this on the `div` tag?

Reply:
Totally!